DashAI/front/src/components/configurableObject/Inputs/FormInputWrapper.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FormTooltip from "../FormTooltip";
+import { Grid } from "@mui/material";
+/**
+ * Generic component that wraps each form input
+ * @param {string} name name of the input to use as an identifier
+ * @param {string} description text to put in a tooltip that helps the user to understand the parameter
+ * @param {React.ReactNode} children the input component to render in the layout
+ *
+ */
+function FormInputWrapper({ name, description, children }) {
+  return (
+    <Grid
+      container
+      direction="row"
+      key={name}
+      sx={{ display: "flex", alignItems: "center" }}
```

Comment:
Why not to use the pre-defined parameters like `alignItems="center"`

```suggestion
      alignItems="center"
```